ydb/core/kqp/ut/pg/kqp_pg_ut.cpp

```diff
@@ -4119,7 +4119,317 @@ Y_UNIT_TEST_SUITE(KqpPg) {
         for (size_t i = 0; i < colNames.size(); i++) {
             UNIT_ASSERT_VALUES_EQUAL(ydbResults.begin()->Getcolumns().at(i).Getname(), colNames[i]);
         }
-   }
+    }
+
+    Y_UNIT_TEST(CheckSetConfig) {
```

Comment:
I'd suggest moving the test to the neighbour file pg_catalog_ut.cpp; It's better to avoid putting new tests in kqp_pg_ut.cpp, if possible, because its compilation is really slow :(